packages/common/tsconfig.json

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "../../tsconfig.json"
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "noEmit": true
```

Comment:
Since these are just libraries, having them built and bundled by the only app we have, `host`? Unless we are releasing these as separate packages (which we very well might do one day) no point in emiting anything.. I had hoped it would speed up tsc but it doesn't seem to